components/infobox/commons/infobox_person.lua

```diff
@@ -290,7 +297,11 @@ end
 
 --- Allows for overriding this functionality
 function Person:calculateEarnings(args)
-	return 0
+	_totalEarnings, _earnings = Earnings.calculateForPlayer{
```

Comment:
It looks to me like this function only has one return value? (PS: it's lacking return value docs)

Reply:
yeah i wanted to add docs to the earnings module this week
have mandatory yearly employee training stuff this week, so only on tablet during that (my 2nd monitor died last week)

Reply:
👍

Reply:
Why not use ApiDoc?

Reply:
adjusted

Reply:
Adjusted it further

Reply:
```suggestion
	_totalEarnings, _earningsPerYear = Earnings.calculateForPlayer{
```